src/helpers/partnersReducer.ts

```diff
@@ -53,6 +53,9 @@ export const partnersReducer = (
                 companyName: action.payload,
             }
         case partnersActions.UPDATE_BUSINESS_FIELD:
+            console.log('action', [...state.businessField])
+            console.log('action', action.payload)
+
             let newBusinessField = [...state.businessField]
```

Comment:
Perhaps the grouping should be done when initializing the state with the grouped business fields. This way, they can also be used in other parts, such as the partner view, where we may want to group them as well